### ISSUE TITLE
compile error with Carthage

### ISSUE THREAD

**Issue by Jake6329** [henrinormak/Heimdall#53]
I'm having trouble using the framework with Carthage. I added this line to the file `Cartfile`:

```
github "henrinormak/Heimdall"
```

And the ran `carthage update`, but I got compilation errors. The error message starts with this:

```
*** Fetching Heimdall
*** Checking out Heimdall at "1.1.0"
*** xcodebuild output can be found in /var/folders/xv/4249pm313xzftn13dv3ncb940000gn/T/carthage-xcodebuild.fb2uap.log
*** Building scheme "Heimdall-TVOS" in Heimdall.xcodeproj
/Users/jakelim/git/local/PrivacyGuard/Carthage/Checkouts/Heimdall/Heimdall/Heimdall.swift:40:1: error: expressions are not allowed at the top level
/Users/jakelim/git/local/PrivacyGuard/Carthage/Checkouts/Heimdall/Heimdall/Heimdall.swift:40:5: error: consecutive statements on a line must be separated by ';'
/Users/jakelim/git/local/PrivacyGuard/Carthage/Checkouts/Heimdall/Heimdall/Heimdall.swift:41:5: error: expected declaration
/Users/jakelim/git/local/PrivacyGuard/Carthage/Checkouts/Heimdall/Heimdall/Heimdall.swift:41:16: error: consecutive declarations on a line must be separated by ';'
```

The line 40 in Heimdall.swift starts with the keyword `open` which is introduced in Swift 3. It seems that Heimdall-TVOS target is still configured to use the legacy Swift syntax, and the Xcode build settings show this as in the screenshot below.

<img width="1064" alt="screen shot 2016-10-29 at 5 44 13 pm" src="https://cloud.githubusercontent.com/assets/16357444/19833669/7064fb18-9dff-11e6-97b5-a8d7381bc413.png">

To me, it looks like the build setting is the cause of the problem. I'm not sure if there could be further issues. Can this be fixed?

**Comment by henrinormak** [comment]
Investigating, currently in the xcode-tweaks branch, but will likely merge once sorted.

**Comment by henrinormak** [comment]
Should be fixed in 1.1.2 (i.e master), please verify.
